LayoutTests/platform/ios-wk2/TestExpectations

```diff
@@ -2291,6 +2291,8 @@ webkit.org/b/243515 svg/compositing/outermost-svg-directly-composited-transforme
 webkit.org/b/243515 svg/compositing/outermost-svg-directly-composited-shape-child.html [ Pass ImageOnlyFailure ]
 webkit.org/b/243515 svg/compositing/outermost-svg-directly-composited-group-child.html [ Pass ImageOnlyFailure ]
 
+webkit.org/b/243702 imported/w3c/web-platform-tests/css/cssom-view/position-sticky-root-scroller-with-scroll-behavior.html [ Failure ]
```

Comment:
The bug should represent the unfixed reason this tested needed to be marked a failure, rather than tracking the commit that will make it fail (the bug needs to remain open while this test is still marked as a failure).

Reply:
Made https://bugs.webkit.org/show_bug.cgi?id=261849 to track these tests test failures.